cea/demand/demand_main.py

```diff
@@ -69,60 +71,72 @@ def demand_calculation(locator, weather_path, gv, use_dynamic_infiltration_calcu
     t0 = time.clock()
 
     # weather model
-    weather_data = epwreader.epw_reader(weather_path)[['drybulb_C', 'wetbulb_C', 'relhum_percent', 'windspd_ms', 'skytemp_C']]
+    weather_data = epwreader.epw_reader(weather_path)[
+        ['drybulb_C', 'wetbulb_C', 'relhum_percent', 'windspd_ms', 'skytemp_C']]
 
     building_properties, schedules_dict, date = properties_and_schedule(gv, locator)
 
     # in case gv passes a list of specific buildings to simulate.
-    if gv.simulate_building_list:
-        list_building_names = gv.simulate_building_list
-    else:
+    if buildings == []:
```

Comment:
a more pythonic way of doing this test is `if not buildings` - this has to do with an empty list, as well as `None`, `0`, `""` and `False` being all "falsy", as far as python is concerned.

Reply:
chnaged!